code/modules/events/vent_clog.dm

```diff
@@ -96,3 +96,29 @@
 			foam.set_up(200, get_turf(vent), R)
 			foam.start()
 		CHECK_TICK
+
+//lazy citadel edit
+/datum/round_event_control/vent_clog/aphro
+	name = "Aphrodisiac Backpressure Surge"
+	max_occurrences = 0
+	typepath = /datum/round_event/vent_clog/aphro
+
+/datum/round_event/vent_clog/aphro
+	reagentsAmount = 100
+	var/reagentid = "aphro"
+	var/annoucemsg = "The scrubbers network is experiencing an unexpected surge of pressurized aphrodisiac. Some ejection of contents may occur."
+
+/datum/round_event/vent_clog/aphro/announce()
+	priority_announce(announcemsg, "Atmospherics alert")
+
+/datum/round_event/vent_clog/aphro/start()

```

Comment:
useless override, all you had to do was set randomProbability to 0 and saferChems to list("aphro")